pkg/providers/v1/aws.go

```diff
@@ -5124,6 +5124,12 @@ func nodeNameToIPAddress(nodeName string) string {
 
 func (c *Cloud) nodeNameToInstanceID(nodeName types.NodeName) (InstanceID, error) {
 	if strings.HasPrefix(string(nodeName), rbnNamePrefix) {
+		// depending on if you use a RHEL or Debian based distribution, the hostname on the machine may be either
```

Comment:
In the comment, can you make it clear which distro has which name format?  I think its useful information.

Reply:
Its complicated as it can vary. Upstream Ubuntu uses a non-FQDN, but EKS Ubuntu has a FQDN. I tried to clarify more.